src/porepy/grids/simplex.py

```diff
@@ -70,8 +70,15 @@ def __init__(
 
         assert num_nodes > 2  # Check of transposes of point array
 
-        # Face node relations
+        # Tabulate the nodes in [first, second, third] faces of each triangle
+        # in counterclockwise order
         face_nodes = np.hstack((tri[[0, 1]], tri[[1, 2]], tri[[2, 0]])).transpose()
+
+        # The cell-face orientation is positive if
+        # it coincides with the face orientation from low to high node index
```

Comment:
Again this assumes a certain ordering of `face_nodes`. Please document.

Reply:
We have added a comment in the TriangleGrid docstring and in the code before the definition of face_nodes